lib/bundles/inspec-compliance/target.rb

```diff
@@ -27,7 +26,7 @@ def self.resolve(target, _opts = {})
       # verifies that the target e.g base/ssh exists
       profile = uri.host + uri.path
       Compliance::API.exist?(config, profile)
-      super(target_url(config, profile), config)
+      resolve_next(target_url(config, profile), config)
```

Comment:
Just to clarify this, we should add the behavior to our fetcher class. We are switching from an inheritance-like approach to full fetcher chaining.